net/unbound/Portfile

```diff
@@ -74,8 +75,20 @@ startupitem.create	yes
 startupitem.name	unbound
 startupitem.logfile	/Library/Logs/unbound-startupitem.log
 startupitem.logevents	yes
-startupitem.start	"(${prefix}/sbin/unbound-anchor -a ${prefix}/etc/${name}/root.key 2>&1) || : && (chown ${unbounduser}:${unboundgroup} ${prefix}/etc/${name}/root.key 2>&1) && (${prefix}/sbin/unbound 2>&1)"
-startupitem.stop	"(/bin/kill \$(cat ${prefix}/var/run/${name}/unbound.pid) 2>&1)"
+# the following entries handle multiple instances running in parallel
+startupitem.start	"(${prefix}/sbin/unbound-anchor -a ${prefix}/etc/${name}/root.key 2>&1) || : && (chown ${unbounduser}:${unboundgroup} ${prefix}/etc/${name}/root.key 2>&1) && (for i in ${prefix}/etc/${name}/${name}*conf; do ${prefix}/sbin/unbound -c \$i 2>&1; done)"
+startupitem.stop	"(for i in ${prefix}/var/run/${name}/${name}*pid; do /bin/kill -15 \$(cat \$i) 2>&1; done)"
```

Comment:
Would it be possible for you to put some `\` characters in these to make them readable? It's hard to check what the code does right now.